arctic/chunkstore/date_chunker.py

```diff
@@ -92,7 +92,14 @@ def filter(self, data, range_obj):
         if 'date' in data.index.names:
             return data[range_obj.start:range_obj.end]
         elif 'date' in data.columns:
-            return data[(data.date >= range_obj.start) & (data.date <= range_obj.end)]
+            if range_obj.start and range_obj.end:
```

Comment:
The `DateRange` objects support `OPEN` vs `CLOSED` time bounds.  We make some effort to get this right here:
https://github.com/manahl/arctic/blob/master/arctic/date/_util.py#L112

Would be nice to respect that in ChunkStore.